ports.mk

```diff
@@ -111,15 +111,20 @@ ifdef USE_SPARKLE
 	cp -R $(SPARKLEPATH)/Sparkle.framework $(bundle_name)/Contents/Frameworks/
 endif
 	cp $(srcdir)/icons/scummvm.icns $(bundle_name)/Contents/Resources/
-	cp $(DIST_FILES_DOCS) $(bundle_name)/
+	cp $(DIST_FILES_DOCS) $(bundle_name)/Contents/Resources/
 	cp $(DIST_FILES_THEMES) $(bundle_name)/Contents/Resources/
 ifdef DIST_FILES_NETWORKING
 	cp $(DIST_FILES_NETWORKING) $(bundle_name)/Contents/Resources/
 endif
 ifdef DIST_FILES_ENGINEDATA
 	cp $(DIST_FILES_ENGINEDATA) $(bundle_name)/Contents/Resources/
 endif
-	$(srcdir)/devtools/credits.pl --rtf > $(bundle_name)/Contents/Resources/Credits.rtf
+	$(srcdir)/devtools/credits.pl --rtf > $(bundle_name)/Contents/Resources/AUTHORS.rtf
+	rm $(bundle_name)/Contents/Resources/AUTHORS
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.LGPL >> $(bundle_name)/Contents/Resources/COPYING
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.FREEFONT >> $(bundle_name)/Contents/Resources/COPYING
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.BSD >> $(bundle_name)/Contents/Resources/COPYING
+	rm $(bundle_name)/Contents/Resources/COPYING.LGPL $(bundle_name)/Contents/Resources/COPYING.FREEFONT $(bundle_name)/Contents/Resources/COPYING.BSD
```

Comment:
I am not a lawyer, but perhaps a simple way to comply with the comments made by @sev- [here](https://github.com/scummvm/scummvm/pull/1296#issuecomment-414053875) would be to remove this line and keep all the COPYING files unmodified inside the bundle, and instead create a fifth file, call it something like COPYING-COMBINED, concatenate all the license files into it, and link the Licensing Information menu item to this. In this way the license files would still be distributed in unmodified form along with the binary. Something like [this](https://github.com/angstsmurf/scummvm/commit/62d29e1ed81c2b0c7b9df0a1dfef6186554a2fcc).

An alternative would be to create three new menu items, one for every license file.